### PR TITLE
Create friday.py

### PR DESCRIPTION
**Pull request by aaron-bowman**
Adding friday function. Works local - needs to be "Q'ified".

### CODE REVIEW

File: bot/commands/friday.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+    request = requests.get(giphy_url, params=params)
+    if request.ok:
+        gif_url = random.choice(request.json()['data'])['images']['downsized']['url']
+        logger.debug("Successfully retrieved Gif: {}".format(gif_url))
+        return "Powered by GIPHY: {}".format(gif_url)
+    else:
+        logger.error("Unable to get Gif from Giphy: {}".format(request.content))
+        return False
+
```
Comment (LEXmono) [inline bot/commands/friday.py]: 2 blank lines before functions per PEP8

File: bot/commands/friday.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+def friday(query=[], user=None):
+	if today == 4:
+			response="http://i.imgur.com/MXaLbQk.gif"
```
Comment (LEXmono) [inline bot/commands/friday.py]: Spaces before/after '=' please.

File: bot/commands/friday.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	else:
+			response = get_giphy('fml')
+	return response
+
+if __name__ == '__main__':
```
Comment (LEXmono) [inline bot/commands/friday.py]: We can kill this once you're done testing.

File: bot/commands/friday.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    request = requests.get(giphy_url, params=params)
+    if request.ok:
+        gif_url = random.choice(request.json()['data'])['images']['downsized']['url']
+        logger.debug("Successfully retrieved Gif: {}".format(gif_url))
+        return "Powered by GIPHY: {}".format(gif_url)
+    else:
+        logger.error("Unable to get Gif from Giphy: {}".format(request.content))
+        return False
+
+def friday(query=[], user=None):
```
Comment (LEXmono) [inline bot/commands/friday.py]: Before this function add the following after testing to "Qify it"

`@command_map.register_command()`

File: bot/commands/friday.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import sys
+import os
```
Comment (LEXmono) [inline bot/commands/friday.py]: I really need to alphabetize all the imports in Q. One day maybe. 

You will need to `from bot.command_map import command_map` here for Q to be able to register this command and call it.

File: bot/commands/friday.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    if request.ok:
+        gif_url = random.choice(request.json()['data'])['images']['downsized']['url']
+        logger.debug("Successfully retrieved Gif: {}".format(gif_url))
+        return "Powered by GIPHY: {}".format(gif_url)
+    else:
+        logger.error("Unable to get Gif from Giphy: {}".format(request.content))
+        return False
+
+def friday(query=[], user=None):
+	if today == 4:
```
Comment (LEXmono) [inline bot/commands/friday.py]: Can we add a logger.debug() here outputting the current day and any other information the function uses so we have that in the logs?

Something like:
`logger.debug("Running friday.py.\ntoday: {}\nQuery: {}\nUser: {}".format(today, " ".join(query), user)`

**Comment by aaron-bowman** [comment]
Making updates

**Pull request closed without merging**